### PR TITLE
Fixed loss of Japanese charactes in the end of lines which was introduced by hyphenation fixes

### PR DESCRIPTION
**Pull request by artxp**
https://trello.com/c/9lkrdGZO/622-some-japanese-characters-missing-in-paragraphs

### CODE REVIEW

**Comment by artxp** [comment]
paired with https://github.com/area9innovation/innovation/pull/933

**Comment by artxp** [comment]
I broke Japanese functionality when fixed Allowing breaks after some symbols in English.

According to https://github.com/area9innovation/flow9/blob/master/lib/text/localization.flow#L135 we have to differentiate approaches for breaking symbols whether it's CJKLang() or not. So for Japanese (wich is CJKLang) I restored the previous approach.

File: lib/form/paragraph.flow [hunk 1]
```diff
@@ -681,9 +681,12 @@ breakTextFragment(
 						PositionAllowBreakAfter() : {
 							toInspectingElementIfNecessary(indexing,
 								indexing + keyPosRelative,
-								makeSpace(getCharAt(text, keyPos),
-									style
-								)
+								// in case of CJKLang we don't show the break symbol
```
Comment (ykuzmenko73) [inline lib/form/paragraph.flow]: Extract the knowledge about the language's specifics from paragraph.flow into separate file related to languages and use it's functions here.
The function could for example returns special data type with all required field to be used here

Comment (artxp) [reply lib/form/paragraph.flow]: @ykuzmenko73 addressed

**Comment by artxp** [comment]
[Some Japanese characters missing in paragraphs](https://trello.com/c/9lkrdGZO/622-some-japanese-characters-missing-in-paragraphs)

**Pull request merged**